.circleci/config.yml

```diff
@@ -42,6 +42,11 @@ commands:
             equal: [ "386", << parameters.arch >> ]
           steps:
             - run: echo 'export RACE="-race"' >> $BASH_ENV
+      - when:
+          condition:
+            equal: [ windows, << parameters.os >> ]
+          steps:
+            - run: echo 'export CGO_ENABLED=1' >> $BASH_ENV
```

Comment:
Why does this need to be added, when this was previously working?

Reply:
I don't know, I only know that the job complains about "cgo must be enabled for race check" if you don't set this. Maybe the new go-env has CGO_ENABLED set to zero?